silkworm/common/log.hpp

```diff
@@ -27,31 +28,40 @@ enum LogLevels {
     LogTrace, LogDebug, LogInfo, LogWarn, LogError, LogCrit, LogNone
```

Comment:
What does LogNone mean?

Reply:
Just that.  Useful for creating a logger that is silent until turned on, or made silent later.

Reply:
Hmm. It might be confusing to the caller. I'd expect `SILKWORM_LOG(LogNone) << "hello" << endl;` not to print anything, but it's actually above LogCrit.

Reply:
Yes. If we order verbosity in your direction it goes at the other end.

Reply:
I'd remove LogNone altogether to avoid potential confusion.

Reply:
LogNone is the only way to turn off logging.